paasta_tools/cli/cmds/rerun.py

```diff
@@ -85,6 +88,21 @@ def add_subparser(subparsers):
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+    rerun_parser.add_argument(
+        '-t', '--rerun-type',
```

Comment:
What do you think if this was simplified to a single flag called `--rerun-dependent-jobs-too` ?